content/reference/sql/sql-support/expressions-functions-operators.md

```diff
@@ -431,6 +431,7 @@ Most functions are simple to implement. If you need one that isn't implemented,
 | `ST_NumInteriorRing()`            | ❌         |                                                                   |
 | `ST_NumPoints()`                  | ❌         |                                                                   |
 | `ST_Overlaps()`                   | ❌         |                                                                   |
+| `ST_Perimeter()`                  | ✅         | Geodetic not yet supported                                        |
```

Comment:
Maybe add a note. Not supported in MySQL. Follows Postgres standard with a link to the Postgres docs?